### ISSUE TITLE
Test.pm needs update

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-engine#699]
[Test.pm] (here linked to develop branch) refer to `Mail::RFC822::Address` which is not used anywhere in the Engine code.

```
info( DEPENDENCY_VERSION => { name => 'Mail::RFC822::Address', version => $Mail::RFC822::Address::VERSION } );
```

Could that be removed? Anything else in that list?

[Test.pm]: https://github.com/zonemaster/zonemaster-engine/blob/develop/lib/Zonemaster/Engine/Test.pm

**Comment by mattias-p** [comment]
Good digging. Yeah, Mail::RFC822::Address was replaced with Email::Valid in #547 for packaging reasons.

This entire list of dependencies should match what's listed in Makefile.PL as `requires` and `recommends`.

**Comment by matsduf** [comment]
Resolved by #700

**Comment by matsduf** [comment]
Resolved by #700.
